style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: flex;
+  height: 100vh;
+  text-align: center;
+  align-items: center;
+  justify-content: space-around;
+}
+.cards {
+  width: 25%;
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 300px;
+}
+.container .cards .face1 img {
+  margin: 15px;
+}
+.container .cards .face1 img:hover {
+  width: 105px;
+  transition: 0.2s;
+}
+.container .cards .face1 {
+  background: rgb(177, 157, 157);
+  width: 100%;
+  z-index: 1;
+  transform: translateY(100px);
+  height: 33vh;
```

Comment:
grp this properties @1khansaad

Reply:
i have changed the code please check it again